website/docs/r/application_gateway.html.markdown

```diff
@@ -204,6 +204,8 @@ A `backend_http_settings` block supports the following:
 
 * `request_timeout` - (Required) The request timeout in seconds, which must be between 1 and 86400 seconds.
 
+* `host_name` - (Optional) Host header to be sent to the backend servers. Cannot be set if `pick_host_name_from_backend_address` is set to `true`.
```

Comment:
```suggestion
* `host_name` - (Optional) Host header to be sent to the backend servers. Should be skipped if `pick_host_name_from_backend_address` is set to `true`.
```

Reply:
I don't think we should make this change. `Cannot be set` is more inline with the existing documentation so is fine as is as well as is definitive in it should not be set.